### PR TITLE
This branch is to create core functionality add and remove with edit on Todo list app

### PR DESCRIPTION
**Pull request by steveWDamesJr**
Feat:
   - Remove hardcoded tasks from  array object.
   - Implement a function for adding a new task (add a new element to the array).
   - Implement a function for deleting a task (remove an element from the array).
   - Implement a function for editing task descriptions.
   - Default property of completed new tasks set to false 
   - Create JavaScript file for new feature.
   - Changes to List tasks saved in local storage.

Debug:
  - Correct linter errors

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,37 +1,118 @@
...
+function deleteTodo(key) {
+  const index = userInput.findIndex((item) => item.id === Number(key));
+  const todo = {
+    deleted: true,
+    ...userInput[index],
+  };
+  userInput = userInput.filter((item) => item.id !== Number(key));
+  renderTodo(todo);
+}
+
```
Comment (andersonlebon) [inline src/index.js:64]: - [x] Kindly note that when I am deleting one task sometimes is cleaning the arrays in your localstorage. Kindly fix this issue. check the screen video below 👇 

https://user-images.githubusercontent.com/65068771/151587526-39af9129-c95c-4afb-99eb-86b688545aec.mp4

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @steveWDamesJr 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (steveWDamesJr) [reply src/index.js:64]: Thank you for your reply.
I have made the required changes. 
All test pass.

https://user-images.githubusercontent.com/81635839/151604003-238a1a93-112c-4f7a-9a21-b37d403f3007.mov

**Review by ZeenatLawal — APPROVED** [review]
Hi @steveWDamesJr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**